books/free-programming-books-langs.md

```diff
@@ -378,12 +378,14 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 ### Bash
 
 * [Advanced Bash-Scripting Guide](http://tldp.org/LDP/abs/html/) - M. Cooper (HTML)
+* [Bash Cook book](https://drive.google.com/file/d/15LVUS7KI6JB9OmZSrkNiwN1DLHaz8ttS/view) - Carl Albing & JP Vossen (PDF)
```

Comment:
Looks like this is a Paid book. Please provide only resources which are permitted for Free use by the license